com.unity.shadergraph/Editor/Generation/Processors/Generator.cs

```diff
@@ -88,6 +90,7 @@ public ActiveFields GatherActiveFieldsFromNode(AbstractMaterialNode outputNode,
         void BuildShader()
         {
             var activeNodeList = Pool.ListPool<AbstractMaterialNode>.Get();
+            bool ignoreActiveState = (m_Mode == GenerationMode.Preview);  // for previews, we ignore node active state
```

Comment:
Each node's active state is set based on the active target, but for previews we use the preview target, so the active state doesn't really apply to them.